admin_app_2/src/components/NavBar.tsx

```diff
@@ -0,0 +1,217 @@
+"use client";
+import { appColors, appStyles, sizes } from "@/utils";
+import MenuIcon from "@mui/icons-material/Menu";
+import { Box } from "@mui/material";
+import AppBar from "@mui/material/AppBar";
+import Avatar from "@mui/material/Avatar";
+import IconButton from "@mui/material/IconButton";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import Tooltip from "@mui/material/Tooltip";
+import Typography from "@mui/material/Typography";
+import Link from "next/link";
+import { usePathname, useRouter } from "next/navigation";
+import * as React from "react";
+import logowhite from "../../../docs/images/logo-light.png";
+import { Layout } from "./Layout";
+
+const pages = [
+  { title: "Playground", path: "/playground" },
+  { title: "Manage Content", path: "/content" },
+  { title: "Dashboard", path: "/dashboard" },
+];
+
+const settings = ["Logout"];
```

Comment:
This button doesn't work

Reply:
Yep - nothing in NavBar works. Its just a placehoder